### DISCUSSION TITLE
How to write a GET API simplest method in Flutter?

### DISCUSSION THREAD

**Discussion by Ali-Murtaza143** [community/community#137180] [Programming Help]
### Select Topic Area

General

### Body

I want to know simplest method for API

**Comment by ShahzainAhmed** [comment]
GET API:

 static getApi() async {
    final uri = Uri.parse("https://reqres.in/api/users");
    var response = await http.get(uri);

    if (response.statusCode == 200) {
      print('Response data: ${response.body}');
    } else {
      print('Failed to load data. Status code: ${response.statusCode}');
    }
  }

**Comment by ebndev** [comment]
Hi @Ali-Murtaza143 & @ShahzainAhmed, thanks for being a part of the GitHub Community!

While we appreciate you wanting to contribute to Discussions, please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
